config/initializers/sentry.rb

```diff
@@ -1,10 +1,6 @@
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = Rails.application.config.x.sentry_dsn
-  config.environments = %w[production staging]
-  config.current_environment = Rails.env
+  config.enabled_environments = %w[production staging]
+  config.environment = Rails.env
   config.release = ENV.fetch('BUILD_HASH', nil)
-  config.async = lambda { |event|
-    SentryJob.perform_later(event)
```

Comment:
Sentry has its own background workers, and async is being deprecated (https://github.com/getsentry/sentry-ruby/issues/1522)